ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -3330,6 +3330,55 @@ def test_paddle_tensor_squeeze_(
     )
 
 
+# stack
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="stack",
+    dtype_value=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric").filter(
+            lambda x: "bfloat16" not in x[0]
+        ),
+        shape=st.shared(helpers.get_shape(), key="shape"),
+        num_arrays=st.shared(helpers.ints(min_value=2, max_value=4), key="num_arrays"),
+    ),
+    axis=helpers.get_axis(
+        shape=st.shared(helpers.get_shape(), key="shape"),
+        allow_neg=True,
+        force_int=True,
+    ),
+)
+def test_paddle_tensor_stack(
+    dtype_value,
+    axis,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+    backend_fw,
+):
+    input_dtype, data = dtype_value
+    x = Tensor(data[0])
```

Comment:
Explicit type casting shouldn't be done here. The value of data should be an array if I'm correct.

Reply:
@bipinKrishnan  Actually it requrires a list of paddle `Tensor` .. As defined [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/stack_en.html#stack) , the right way to provide input to the stack function..I have written a custom function to generate the necessary input for the tests